tsconfig.json

```diff
@@ -7,139 +7,14 @@
-        "src/lib/models/reflections/declaration.ts",
-        "src/lib/models/reflections/index.ts",
-        "src/lib/models/reflections/parameter.ts",
-        "src/lib/models/reflections/project.ts",
-        "src/lib/models/reflections/signature.ts",
-        "src/lib/models/reflections/type-parameter.ts",
-        "src/lib/models/sources/directory.ts",
-        "src/lib/models/sources/file.ts",
-        "src/lib/models/sources/index.ts",
-        "src/lib/models/types/abstract.ts",
-        "src/lib/models/types/index.ts",
-        "src/lib/models/types/intrinsic.ts",
-        "src/lib/models/types/reference.ts",
-        "src/lib/models/types/reflection.ts",
-        "src/lib/models/types/string-literal.ts",
-        "src/lib/models/types/tuple.ts",
-        "src/lib/models/types/type-parameter.ts",
-        "src/lib/models/types/union.ts",
-        "src/lib/models/types/unknown.ts",
-        "src/lib/output/components.ts",
-        "src/lib/output/events.ts",
-        "src/lib/output/helpers/compact.ts",
-        "src/lib/output/helpers/if-cond.ts",
-        "src/lib/output/helpers/if-signature.ts",
-        "src/lib/output/helpers/wbr.ts",
-        "src/lib/output/models/NavigationItem.ts",
-        "src/lib/output/models/UrlMapping.ts",
-        "src/lib/output/plugins/AssetsPlugin.ts",
-        "src/lib/output/plugins/JavascriptIndexPlugin.ts",
-        "src/lib/output/plugins/LayoutPlugin.ts",
-        "src/lib/output/plugins/MarkedLinksPlugin.ts",
-        "src/lib/output/plugins/MarkedPlugin.ts",
-        "src/lib/output/plugins/NavigationPlugin.ts",
-        "src/lib/output/plugins/PrettyPrintPlugin.ts",
-        "src/lib/output/plugins/TocPlugin.ts",
-        "src/lib/output/plugins/index.ts",
-        "src/lib/output/renderer.ts",
-        "src/lib/output/theme.ts",
-        "src/lib/output/themes/DefaultTheme.ts",
-        "src/lib/output/themes/MinimalTheme.ts",
-        "src/lib/output/utils/resources.ts",
-        "src/lib/output/utils/resources/helpers.ts",
-        "src/lib/output/utils/resources/stack.ts",
-        "src/lib/output/utils/resources/templates.ts",
-        "src/lib/utils/component.ts",
-        "src/lib/utils/events.ts",
-        "src/lib/utils/fs.ts",
-        "src/lib/utils/index.ts",
-        "src/lib/utils/loggers.ts",
-        "src/lib/utils/options/declaration.ts",
-        "src/lib/utils/options/help.ts",
-        "src/lib/utils/options/index.ts",
-        "src/lib/utils/options/options.ts",
-        "src/lib/utils/options/readers/arguments.ts",
-        "src/lib/utils/options/readers/index.ts",
-        "src/lib/utils/options/readers/tsconfig.ts",
-        "src/lib/utils/options/readers/typedoc.ts",
-        "src/lib/utils/options/sources/component.ts",
-        "src/lib/utils/options/sources/index.ts",
-        "src/lib/utils/options/sources/typescript.ts",
-        "src/lib/utils/plugins.ts",
-        "src/index.ts"
+    "include": [
+        "src/**/*.ts"
+    ],
+    "exclude": [
```

Comment:
Looks like it's been reported https://github.com/TypeStrong/grunt-ts/issues/381.